mongo-stream.js

```diff
@@ -66,10 +66,16 @@ class MongoStream {
     }
   }
 
-  async dumpCollections(collections) {
+  async dumpCollections(collections, ignoreResumeTokens = false) {
```

Comment:
seems redundant to apply a default value to the param when you already guarantee the value in server.js

Reply:
You could use this with a different front end implementation.